app/assets/javascripts/discourse/app/widgets/topic-admin-menu.js

```diff
@@ -106,6 +107,16 @@ createWidget("topic-admin-menu-button", {
     }
 
     this.state.position = position;
+
+    next(() => {
+      let menuButtons = document.querySelectorAll(
+        ".topic-admin-popup-menu button"
+      );
+
+      if (menuButtons && menuButtons[0]) {
+        menuButtons[0].focus();
+      }
+    });
```

Comment:
Instead of `next()` you can take advantage of the recently added callback - `didRenderWidget`! 😃 For example see:

https://github.com/discourse/discourse/pull/10496/files#diff-796ca2d406c1ba53a991622c8b6c2e3cR245